src/ko/newtoki/src/eu/kanade/tachiyomi/extension/ko/newtoki/NewToki.kt

```diff
@@ -182,8 +182,10 @@ abstract class NewToki(
     private fun mangaDetailsParseWithTitleCheck(manga: SManga, document: Document) =
         mangaDetailsParse(document).apply {
             // TODO: don't throw when there is download folder rename feature
-            if (manga.description.isNullOrEmpty() && manga.title != title) {
-                throw Exception(titleNotMatch(title))
+            if (manga.description.isNullOrEmpty() && title.removeSuffix("…") !in manga.title) {
+                val tmp = title.split("…")[0]
+                if (!manga.title.contains(tmp))
+                    throw Exception(titleNotMatch(title))
```

Comment:
```suggestion
                throw Exception(titleNotMatch(title))
```

You also need to revert this part to the original.